Mobile/Android/Templates/Release_1.9.2/1510-Edit_in_public_links.md

```diff
@@ -0,0 +1,35 @@
+###  Edit permission in public shared links 
+
+#### Pr: https://github.com/owncloud/android/pull/1510
+
+Devices:
+
+Server: 
+
+
+---
+
+ 
+| TestID | Test Case | Steps | Expected Result | Result | Related Comment |
+| :----: | :-------- | :---- | :-------------- | :----- | :-------------- |
+|**View**||||||
+|1|Folders|1. Long press to share a folder by link<br>| A switch button with the option "Allow editing" appears|||
+|2|Files|1. Long press to share a folder by link<br>| A switch button with the option "Allow editing" does not appear|||
+|**Orientation**||||||
+|3|Portrait|1. Long press to share a folder by link<br>| A switch button with the option "Allow editing" appears correctly|||
+|4|Landscape|1. Long press to share a folder by link<br>| A switch button with the option "Allow editing" appears correctly|||
+|**Actions**||||||
+|5|Allow - Upload|1. Long press to share a folder by link.<br>2. Enable "Allow Editing"<br>3. Copy the link and paste in a browser<br>4. Upload content to the folder| 2. Check also in server that is enabled<br>4. Content is upload. Check in app|||
+|6|Allow - New folder|1. Long press to share a folder by link.<br>2. Enable "Allow Editing"<br>3. Copy the link and paste in a browser<br>4. Create a subfolder|2. Check also in server that is enabled<br>4. Subfolder is created and content can be uploaded on it. Check in app|||
+|7|Disabling Allow|1. From 6, disable "Allow Editing"<br>2. Copy the link and paste in a browser|1. Check also in server that is disabled<br>3. No options for editing are available|||
+|**External actions**||||||
+|8|Allow |1. In server, share a folder by link and enable "Allow editing".| Check in app that "Allow editing" is also enabled|||
```

Comment:
Test case 6 is when you enable the in mobile device and check that it works.
Test case 8 is when you enable from server and check that the option appears enabled in mobile app.